sign/src/sign.py

```
@@ -30,19 +30,42 @@
 app = Flask(__name__)
 
 def get_local_ip():
+    # Try to get the IP address by connecting to a public DNS server
     s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
     try:
-        s.connect(('192.255.255.255', 1))
+        # Try connecting to Google's DNS server
+        s.connect(('8.8.8.8', 80))
         IP = s.getsockname()[0]
     except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)